<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>B_Theta_form</class>
 <widget class="QWidget" name="B_Theta_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1351</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCustomPlot" name="energy_angle_Plot" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>531</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="equation_angle_plot" native="true">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>110</y>
     <width>751</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Energy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Eqation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>730</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Angle</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="B_perp_spinBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>630</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QSlider" name="B_perp_Slider">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>640</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="B_par_Slider">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>590</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>60000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="B_par_spinBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>590</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>60.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>590</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>B par</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>630</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>B perp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>690</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>B tot</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nu_i_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>750</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="n_g_lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>790</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="n_j_lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>750</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nu_j_lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>750</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nu_g_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>790</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replot_Button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>780</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>REPLOT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="n_i_lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>750</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="n_f_lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>790</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nu_f_lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>790</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="qw_input_TextEdit" name="EnergiesTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Energies</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>780</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Transitions</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>540</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Transitions</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>570</y>
     <width>421</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PrintEquationButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>520</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Print Equation</string>
   </property>
  </widget>
  <widget class="QSlider" name="btot_slider">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>690</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="angle_Slider">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>740</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>90000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="btot_SpinBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>680</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="angle_SpinBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>730</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="PrintLevelsButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>520</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Print Levels</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Indep_checkBox">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>570</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Independence</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбор резонансного наклонного магнитного поля.</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qw_input_TextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">qw_input_textedit.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>Plot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
